library/core/src/ptr/non_null.rs

```diff
@@ -480,7 +478,7 @@ impl<T> NonNull<[T]> {
     ///
     /// (Note that this example artificially demonstrates a use of this method,
     /// but `let slice = NonNull::from(&x[..]);` would be a better way to write code like this.)
-    #[unstable(feature = "nonnull_slice_from_raw_parts", issue = "71941")]
+    #[stable(feature = "nonnull_slice_from_raw_parts", since = "1.64.0")]
     #[rustc_const_unstable(feature = "const_nonnull_slice_from_raw_parts", issue = "71941")]
```

Comment:
Maybe it's better if we close #71941 and move tracking of the const part to the const_slice_from_raw_parts issue.
```suggestion
    #[rustc_const_unstable(feature = "const_slice_from_raw_parts", issue = "67456")]
```

Reply:
Sounds good! Updated as suggested (sorry for the delay, I was sick these days).